src/main/java/uk/gov/companieshouse/service/dissolution/DissolutionPatcher.java

```diff
@@ -107,6 +116,24 @@ private void addDirectorApproval(String userId, DissolutionPatchRequest body, Di
         director.setDirectorApproval(approval);
     }
 
+    private void updateSignatory(DissolutionPatchDirectorRequest body, Dissolution dissolution, String officerId) throws DissolutionNotFoundException, DirectorNotFoundException {
+        DissolutionDirector director = this.findDirector(officerId, dissolution);
+
+        if (director == null) {
```

Comment:
Should director be Optional< DissolutionDirector >?

Reply:
Removing this check, as when director is not present it will result in an exception.